services/publishing/base-repo/security/policy.json

```diff
@@ -0,0 +1,90 @@
+{
+    "policies": [
+        {
+            "group": "*",
+            "user": "*",
+            "contexts": {},
+            "object": "values/*",
+            "action": "*",
+            "effect": "allow"
+        },
+        {
+            "group": "*",
+            "user": "*",
+            "contexts": {
+                "*": "*"
+            },
+            "object": "values/*",
+            "action": "*",
+            "effect": "allow"
+        },
+        {
+            "group": "*",
+            "user": "*",
+            "contexts": {
+                "tweek_editor_user": "self"
+            },
+            "object": "values/*",
+            "action": "*",
+            "effect": "allow"
+        },
+        {
+            "group": "externalapps",
+            "user": "admin-app",
+            "contexts": {},
+            "object": "repo/keys/*",
+            "action": "write",
+            "effect": "allow"
+        },
+        {
+            "group": "externalapps",
+            "user": "admin-app",
+            "contexts": {},
+            "object": "repo",
+            "action": "read",
+            "effect": "allow"
+        },
+        {
+            "group": "externalapps",
+            "user": "admin-app",
+            "contexts": {},
+            "object": "repo/policies",
+            "action": "read",
+            "effect": "allow"
+        },
+        {
+            "group": "externalapps",
+            "user": "admin-app",
+            "contexts": {},
+            "object": "repo/policies",
+            "action": "write",
+            "effect": "allow"
+        },
+        {
+            "group": "externalapps",
+            "user": "admin-app",
+            "contexts": {},
+            "object": "repo/tags",
+            "action": "write",
+            "effect": "allow"
+        },
+        {
+            "group": "externalapps",
+            "user": "admin-app",
+            "contexts": {},
+            "object": "repo/schemas",
+            "action": "write",
+            "effect": "allow"
+        },
+        {
+            "group": "externalapps",
```

Comment:
I think we can keep just the last permission for admin-app as it basically allows everything